databroker/core.py

```diff
@@ -760,7 +760,8 @@ def read_partition(self, partition):
                             datum_ids |= set(datum_page['datum_id'])
                 if fill == 'yes':
                     self._fill(event)  # in place (for now)
-                payload.append(('event', event))
+                event_page = event_model.pack_event_page(event)
```

Comment:
thought: this could also be done by changing interlace_event_pages to interlace_event_page_chunks. But that may require more work, maybe it is better to do that later.

Reply:
Yeah, I just did the dumbest/simplest thing possible. There are a couple moving pieces involved in actually doing this efficiently.